test-network-function/identifiers/identifiers.go

```diff
@@ -299,13 +306,24 @@ the same hacks.'`),
 		Type:       informativeResult,
 		Remediation: `In most cases, Pod's should not specify their host Nodes through nodeSelector or nodeAffinity.  However, there are
 cases in which CNFs require specialized hardware specific to a particular class of Node.  As such, this test is purely
-informative,  and will not prevent a CNF from being certified.  However, one should have an appropriate justification as
+informative, and will not prevent a CNF from being certified. However, one should have an appropriate justification as
 to why nodeSelector and/or nodeAffinity is utilized by a CNF.`,
 		Description: formDescription(TestPodNodeSelectorAndAffinityBestPractices,
 			`ensures that CNF Pods do not specify nodeSelector or nodeAffinity.  In most cases, Pods should allow for
 instantiation on any underlying Node.`),
 	},
 
+	TestPodAntiAffinityBestPractices: {
+		Identifier: TestPodAntiAffinityBestPractices,
+		Type:       informativeResult,
+		Remediation: `In most HA cases, Pod podAntiAffinity rule should be specified when replica value is set during pod instantiation. 
+		However, there are cases in which CNFs require specialized hardware specific to a particular class of Node.  As such,this test is purely informative
```

Comment:
This sentence does not apply here. It's for node selector/affinity. Maybe get ride of the rest too. This is a pretty firm rule for HA, unlike the node selector check.

Reply:
how about this:  
"In high availability case, podAntiAffinity rule should be specified for pod instantiation if pod replica value is set more than 2 .

Reply:
i would say pod scheduling instead of pod instantiation.